tests/unit/knockout/combo/tests.html

```diff
@@ -332,13 +348,24 @@
 			$element.trigger(mouseUp);
 		}
 
+		function typeInInput(key, element) {
```

Comment:
These methods should be in the `TestUtil`. I believe there is already a method there defined for the same thing.